presto-main/src/main/java/com/facebook/presto/operator/scalar/StringFunctions.java

```diff
@@ -815,4 +815,76 @@ public static Slice toUtf8(@SqlType("varchar(x)") Slice slice)
     {
         return slice;
     }
+
+    @Description("suffix starting at given index")
+    @ScalarFunction("substr")
+    @SqlType(StandardTypes.VARBINARY)
+    public static Slice varbinarySubstr(@SqlType(StandardTypes.VARBINARY) Slice utf8, @SqlType(StandardTypes.BIGINT) long start)
+    {
+        if ((start == 0) || utf8.length() == 0) {
```

Comment:
why there's a need for brackets around `start` variable? is it a style guide thing?

Reply:
I understand your point. Line#216 does the same. Do you want me to update Line#216 as well?

Reply:
I understand that it's similar to another line. But let's remove the bracket here. Being inconsistent in a single line is definitely the worst.